### PR TITLE
Add digasId for shows

### PR DESCRIPTION
**Pull request by santi**

### CODE REVIEW

File: src/models/show.model.js [hunk 1]
```diff
@@ -55,6 +55,10 @@ export default function (sequelize, DataTypes) {
             type: DataTypes.STRING(5),
             allowNull: false,
             defaultValue: 'no'
+        },
+        digasId: { // For use with LDAP
```
Comment (essoen) [inline src/models/show.model.js]: Does this have anything to do with LDAP?
Aren't we switching to AD?

**Review by essoen — CHANGES_REQUESTED** [review]

File: src/models/show.model.js (hunk 1 above)
Comment (santi) [reply src/models/show.model.js]: Nothing to do with LDAP, "Digas" is the program used by radio journalists to manage the archive of podcasts and ondemand streams.

Comment (essoen) [reply src/models/show.model.js]: Then why the comment:
> // For use with LDAP

? =)

Comment (santi) [reply src/models/show.model.js]: Opps, too much copy paste

**Review by essoen — APPROVED** [review]

**Pull request merged**